src/game.h

```diff
@@ -1151,6 +1039,8 @@ class game
         /** How far the tileset should be zoomed out, 16 is default. 32 is zoomed in by x2, 8 is zoomed out by x0.5 */
         int tileset_zoom;
 
+        liquid_handler liquid_handling;
```

Comment:
There's no point keeping this in `game` as an aggregate: `game` doesn't make any use of it (its environment does).

Reply:
so even just putting in a namespace then?
all it is is a bunch of functions.

Reply:
Yes, exactly this [I was too slow writing my comment below].